tests/dashboards/api_tests.py

```diff
@@ -340,6 +340,31 @@ def test_get_dashboards_favorite_filter(self):
                 expected_model.dashboard_title == data["result"][i]["dashboard_title"]
             )
 
+    @pytest.mark.usefixtures("create_dashboards")
+    def test_get_current_user_favorite_status(self):
+        """
+        Dataset API: Test get current user favorite stars
+        """
+        admin = self.get_user("admin")
+        users_favorite_ids = [
+            star.obj_id
+            for star in db.session.query(FavStar.obj_id)
+            .filter(
+                and_(FavStar.user_id == admin.id, FavStar.class_name == "daashboard")
```

Comment:
Typo `daashboard` ? Also, have you tested this on existing fav stars? I think the old queries used `Dashboard` as `class_name` for dashboards, but `slice` for slices. 🤦 

<img src="https://user-images.githubusercontent.com/335541/97774869-aa952c00-1b18-11eb-884a-975ad68ae396.png" width="500" />

Might be good to put these two in a `Enum` if you don't mind.

Reply:
This seems wrong, thanks catching. Not sure how these tests are passing, I’ll dig into this and and make sure these can actually fail.